src/tasks/harbor/harbor.ts

```diff
@@ -98,29 +105,83 @@ function setAuth(secret): void {
   robotApi.setDefaultAuthentication(bearerAuth)
 }
 
+async function getProjectId(team: string): Promise<string> {
+  const project = (await doApiCall(errors, `Get project for team ${team}`, () =>
+    projectsApi.getProject(team),
+  )) as Project
+  if (!project) return ''
+  const projectId = `${ensure(project.projectId)}`
+  return projectId
+}
+
```

Comment:
this whole function has no right to exist as it does nothing. Please just get the projectId somewhere in the top of main and inject it to the methods that need it.

Reply:
done